src/tasks/intern.ts

```diff
@@ -1,26 +1,46 @@
 import global from '@dojo/shim/global';
+import { assign } from '@dojo/core/lang';
 
 import Node from '../lib/executors/Node';
 import { Config } from '../lib/executors/Executor';
+import { getConfig } from '../lib/node/util';
 
 interface TaskOptions extends grunt.task.ITaskOptions, Partial<Config> {
 	[key: string]: any;
 }
 
+function configure(options: TaskOptions): Promise<{
```

Comment:
`configure` is a weird name for this since its not configuring anything (although I don't suppose it matters since its internal). Also, Intern code typically has things like module-level constants and functions declare below exports.